rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -57,6 +57,8 @@
 
 @ApplicationScoped
 public class LRAService {
+    private static final Pattern LINK_REL_PATTERN = Pattern.compile("(\\w+)=\"([^\"]+)\"|([^\\s]+)");
```

Comment:
This change seems unrelated. Let's keep PR limited to the issue that is being fixed.

Reply:
Ah wait, this is a fixing a performance problem since the code is compiling the pattern on every call so your change is needed, sorry.

Reply:
yes. ok, thanks.